packages/block-library/src/utils/migrate-font-family.js

```diff
@@ -21,17 +16,14 @@ export default function ( attributes ) {
 		return attributes;
 	}
 
-	// Clone first so when we delete the fontFamily
-	// below we're not modifying the original
-	// attributes. Because the deprecation may be discarded
-	// we don't want to alter the original attributes.
-	const atts = cloneDeep( attributes );
-	const fontFamily = atts.style.typography.fontFamily.split( '|' ).pop();
-	delete atts.style.typography.fontFamily;
```

Comment:
I'm not sure I can spot where, in the new version of the code, the `fontFamily` property is `delete`d from `attributes.style.typography` ?